packages/containers/src/actionAPI.js

```diff
@@ -20,8 +20,8 @@ export function getActionsProps(context, ids, model) {
 				);
 			} else {
 				context.store.dispatch(Object.assign({
-					model,
-				}, info.payload));
+					model
+				}, info.payload, { openInNewTab: event.ctrlKey || event.button === 1 }));
```

Comment:
why not `{ event: { ctrlKey: event.ctrlKey, button: event.button } }` ?
so the logic is on the onClick